packages/fetch-and-compile/test/fixture.js

```diff
@@ -0,0 +1,84 @@
+const fs = require("fs");
+const path = require("path");
+
+const mainnetData = {
+  status: "1",
+  message: "OK-Missing/Invalid API Key, rate limit of 1/5sec applied",
+  result: [
+    {
+      SourceCode: fs
+        .readFileSync(path.resolve(__dirname, "./sol/mainnet.sol"))
+        .toString(),
+      ABI: fs
+        .readFileSync(path.resolve(__dirname, "./abi/mainnet.abi.json"))
+        .toString(),
+      ContractName: "UniswapV2Router02",
+      CompilerVersion: "v0.6.6+commit.6c089d02",
+      OptimizationUsed: "1",
+      Runs: "999999",
+      ConstructorArguments:
+        "0000000000000000000000005c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+      EVMVersion: "istanbul",
+      Library: "",
+      LicenseType: "GNU GPLv3",
+      Proxy: "0",
+      Implementation: "",
+      SwarmSource:
+        "ipfs://6dd6e03c4b2c0a8e55214926227ae9e2d6f9fec2ce74a6446d615afa355c84f3"
+    }
+  ]
+};
+const arbitrumData = {
+  status: "1",
+  message: "OK",
+  result: [
+    {
+      SourceCode: fs
+        .readFileSync(path.resolve(__dirname, "./sol/arbitrum.sol"))
+        .toString(),
+      ABI: fs
+        .readFileSync(path.resolve(__dirname, "./abi/arbitrum.abi.json"))
+        .toString(),
+      ContractName: "Storage",
+      CompilerVersion: "v0.8.4+commit.c7e474f2",
+      OptimizationUsed: "1",
+      Runs: "1000",
+      ConstructorArguments:
+        "0000000000000000000000009f20de1fc9b161b34089cbeae888168b44b03461",
+      EVMVersion: "Default",
+      Library: "",
+      LicenseType: "",
+      Proxy: "0",
+      Implementation: "",
+      SwarmSource: ""
+    }
+  ]
+};
+const polygonData = {
+  status: "1",
+  message: "OK-Missing/Invalid API Key, rate limit of 1/5sec applied",
+  result: [
+    {
+      SourceCode: fs
+        .readFileSync(path.resolve(__dirname, "./sol/polygon.sol"))
+        .toString(),
+      ABI: fs
+        .readFileSync(path.resolve(__dirname, "./abi/polygon.abi.json"))
+        .toString(),
+      ContractName: "GrowthVault",
+      CompilerVersion: "v0.8.9+commit.e5eed63a",
+      OptimizationUsed: "1",
+      Runs: "200",
+      ConstructorArguments:
```

Comment:
you should probably get rid of the newline here

Reply:
Is there a way to stop husky from doing this? Every time I create a new commit it goes back lol

Reply:
oh if it is a `prettier` thing then I suppose it is fine